src/hb-aat-layout.cc

```diff
@@ -341,10 +376,12 @@ hb_aat_layout_get_feature_types (hb_face_t                    *face,
 
 /**
  * hb_aat_layout_feature_type_get_name_id:
- * @face: a face object
- * @feature_type: feature id
+ * @face: #hb_face_t to work upon
+ * @feature_type: The #hb_aat_layout_feature_type_t of the requested feature type
  *
- * Return value: Name ID index
+ * Fetches the name ID of the specified feature type in the face's 'name' table.
+ *
+ * Return value: Name ID index of the requested feature type
```

Comment:
I think “index” is redundant, given that it is what “ID” stands for, no?

Reply:
I think ID stands for identifier (here). At https://docs.microsoft.com/en-us/typography/opentype/spec/name they always say "name ID" -- as with "script ID"/"language ID"/etc -- so I was just trying for consistency.

Reply:
Anyway, index here as at best confusing. I’d understand this to mean the index of a name table entry (i.e. the *n*th entry), but here it is just returning name IDs and there can be multiple entries in the name table for the same ID.

Reply:
ID is for identifier.  This is NOT an index.